lambdas/adobe-sign-api/src/agreements-controller.ts

```diff
@@ -26,4 +26,21 @@ export const createSigningUrlAgreement = async (event: APIGatewayProxyEvent): Pr
       return lambdaHandleError(apiError);
     }
   });
+}
+
+export const processWebhook = async (event: SQSEvent): Promise<void> => {
+  try {
+    const service = container.resolve(AgreementService);
+    for (let i = 0; i < event.Records.length; i++) {
+      const webhook = new Webhook(JSON.parse(event.Records[i].body));
+      console.log(`AgreementsController.processWebhook: Processing agreement: ${webhook.agreement.id}`);
+      await service.processWebhook(webhook);
+    }
```

Comment:
Could switch to this instead of old for loops
```suggestion
    for (let record of event.Records) {
      const webhook = new Webhook(JSON.parse(record.body));
      console.log(`AgreementsController.processWebhook: Processing agreement: ${webhook.agreement.id}`);
      await service.processWebhook(webhook);
    }
```

Reply:
Nifty, forgot about this